src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -464,9 +466,22 @@ protected override void Dispose(bool disposing)
         {
```

Comment:
1. Why we need a finalizer? I don't think there are any unmanaged resources that we need to get  rid of?
2. base class - SignatureProvider is an abstract class. We don't need to call its dispose.